docs/workflow.md

```diff
@@ -0,0 +1,143 @@
+(import "manifests/platforms/gke.jsonnet") {
+    config:: import "kubeprod-autogen.json",
+    // Place your overrides here
+}
+```
+
+If we evaluate the manifest using `kubecfg` and filter the output searching for the Let's Encrypt environment, we will find this:
+
+```
+$ kubecfg show kubeprod-manifest.jsonnet | grep -- --default-issuer-name
+        - --default-issuer-name=letsencrypt-prod
+```
+
+To customize BKPR and use the Let's Encrypt staging environment, modify the `kubeprod-manifest.jsonnet` like this:
+
+```
+# Cluster-specific configuration
+(import "manifests/platforms/gke.jsonnet") {
+    config:: import "kubeprod-autogen.json",
+    // Place your overrides here
+    cert_manager+: {
+        letsencrypt_environment:: "staging",
+    }
+}
+```
+
+The previous block of code does the following:
+
+1. Imports the manifest at `manifests/platforms/gke.jsonnet` which describes how to deploy BKPR on GKE.
+1. Under the `cert_manager` key (which implements the `cert-manager` component), overrides the `letsencrypt_environment` property to `staging` from its default value of `prod`.
+
+To ensure the override is working as expected, let's evaluate the manifest again using `kubecfg` and filter the output searching for the Let's Encrypt environment. This time we will see that `cert-manager` will use the staging environment instead of the production environment:
+
+```
+$ kubecfg show kubeprod-manifest.jsonnet | grep -- --default-issuer-name
+        - --default-issuer-name=letsencrypt-staging
+```
+
+Please refer to the [components](components.md) documentation for supported customizations in every component. Also refer to the [jsonnet](https://jsonnet.org) documentation.
+
+### Test changes (only if BKPR is already deployed)
+
+For example, to show the differences between the live state (what is currently running in the Kubernetes cluster) and the local configuration reflects, you can use `kubecfg diff` on the root manifest, like this:
+
+```
+$ kubecfg diff kubeprod-manifest.jsonnet
+```
+
+### Deploy changes
+
+```
+$ kubecfg update --ignore-unknown=true --gc-tag kube_prod_runtime kubeprod-manifest.jsonnet
+```
+
+NOTE: There is a [bug in kubecfg](https://github.com/ksonnet/kubecfg/issues/211) that always requires using `--ignore-unknown=true`, even if this command-line flag defaults to `true`.
+
+### Check-in changes
+
+For example,
+
+```
+$ git add kubeprod-manifest.jsonnet
+$ git commit -m "Use Let's Encrypt staging environment"
+```
+
+Note that the **`kubeprod-autogen.json` file contains sensitive information**, like OAuth2 client and cookies secrets, or credentials for accessing the underlying platform's DNS services. If this information is inadvertently exposed, **it could compromise your Kubernetes cluster or any shared infrastructure**. As a best practice, we recommend explicitly ignoring kubeprod-autogen.json file in your source repository. Best practice is to explicitly ignore `kubeprod-autogen.json` file from your source.
```

Comment:
Repeated line: Best practice is to explicitly ignore `kubeprod-autogen.json` file from your source.